templates/zoom-parts/username-section.tpl

```diff
@@ -14,6 +14,12 @@
     <a id="UsernameUsernameList" class="btn btn-small" target="_blank" href="https://en.wikipedia.org/w/index.php?title=Special%3AListUsers&amp;username={$usernamerawunicode|escape:'url'}&amp;group=&amp;limit=1" onMouseUp="$('#UsernameUsernameList').addClass('btn-visited');">Username list</a>
     <a id="UsernameMainspaceSearch" class="btn btn-small" target="_blank" href="https://en.wikipedia.org/w/index.php?title=Special%3ASearch&amp;profile=advanced&amp;search={$usernamerawunicode|escape:'url'}&amp;fulltext=Search&amp;ns0=1&amp;redirs=1&amp;profile=advanced" onMouseUp="$('#UsernameMainspaceSearch').addClass('btn-visited');">Wikipedia mainspace search</a>
     <a id="UsernameGoogleSearch" class="btn btn-small" target="_blank" href="{$baseurl}/redir.php?tool=google&amp;data={$usernamerawunicode|escape:'url'}" onMouseUp="$('#UsernameGoogleSearch').addClass('btn-visited');">Google search</a>
+    {if $showinfo}
+      <a id="EmailVisit" class="btn btn-small" target="_blank" href="http://{$emailurl}" onMouseUp="$('#EmailVisit').addClass('btn-visited');">Visit email domain</a>
```

Comment:
That's a damn good point actually, this needs to use the redirect tool somehow, as otherwise we do leak data here. We probably want custom logic in there somewhere though, since at the moment it's quite dumb; a new mode flag switching it from "tool + data" to "untrusted domain" or something maybe?